pkg/kv/kvserver/protectedts/ptpb/protectedts.proto

```diff
@@ -88,6 +89,21 @@ message Metadata {
 
 // Record corresponds to a protected timestamp.
 message Record {
+  message SchemaObjectsTarget {
+    // IDs are the descriptor IDs of the schema objects being protected by this
+    // Record. This field will only contain database and table IDs.
+    repeated uint32 ids = 1 [(gogoproto.customname) = "IDs",
+      (gogoproto.casttype) = "github.com/cockroachdb/cockroach/pkg/sql/catalog/descpb.ID"];
+  }
+
+  message TenantsTarget {
+    // IDs are the tenant IDs being protected by this Record.
```

Comment:
[mega nit] It's not the tenant IDs being protected, it's the tenant keyspaces. How about "IDs correspond to the tenant keyspaces being protected by this record"?

Reply:
done.